src/Tizen.NUI.Components/Theme/DefaultTheme.cs

```diff
@@ -108,10 +108,15 @@ public HashSet<ExternalThemeKeyList> GetExternalThemeKeyListSet()
 
                 // Picker
                 (new ExternalThemeKeyList(typeof(Picker), typeof(PickerStyle)))
-                    .AddSelector("/ItemTextLabel/Background", (ViewStyle style, Selector<Color> value) => ((PickerStyle)style).ItemTextLabel.BackgroundColor = value)
-                    .AddSelector("/ItemTextLabel/TextColor", (ViewStyle style, Selector<Color> value) => ((PickerStyle)style).ItemTextLabel.TextColor = value)
-                    .AddSelector("/ItemTextLabel/PixelSize", (ViewStyle style, Selector<float?> value) => ((PickerStyle)style).ItemTextLabel.PixelSize = value)
-                    .AddSelector("/Divider/Background", (ViewStyle style, Selector<Color> value) => ((PickerStyle)style).Divider.BackgroundColor = value),
+                    .Add<Size>("/Size", (ViewStyle style, Size value) => ((ViewStyle)style).Size = value)
+                    .AddSelector<float?>("/ItemTextLabel/PixelSize", (ViewStyle style, Selector<float?> value) => ((PickerStyle)style).ItemTextLabel.PixelSize = value, ControlState.Selected)
+                    .Add<Size>("/ItemTextLabel/Size", (ViewStyle style, Size value) => ((ButtonStyle)style).Text.Size = value)
+                    .AddSelector<Color>("/ItemTextLabel/TextColor", (ViewStyle style, Selector<Color> value) => ((PickerStyle)style).ItemTextLabel.BackgroundColor = value, ControlState.Selected)
+                    .AddSelector("/ItemTextLabel/Background", (ViewStyle style, Selector<Color> value) => ((PickerStyle)style).ItemTextLabel.BackgroundColor = value, ControlState.Selected)
```

Comment:
Please change this as follows.

 .AddSelector("/ItemTextLabel/Background"             ->              .AddSelector("/ItemTextLabel/BackgroundColor"

Reply:
Done